src/gfx/reverse.cpp

```diff
@@ -199,6 +222,22 @@ void reverse() {
 
 		if (options.palSpecType == Options::EXPLICIT && palettes != options.palSpec) {
 			warning("Colors in the palette file do not match those specified with `-c`!");
+			fputs("Colors specified in the palette file:         ...and with `-c`:\n", stderr);
```

Comment:
```suggestion
			fputs("Colors specified in the palette file versus with `-c`:\n", stderr);
```

Reply:
The intention is to align the text into tabular data. How about this, then?

```suggestion
			fputs("Colors specified in the palette file:         ...versus with `-c`:\n", stderr);
```

Reply:
Sure.